cico_setup.sh

```diff
@@ -46,7 +46,7 @@ function install_deps() {
 
 function build() {
   echo 'CICO: Cloning keycloak source code repo'
-  git clone https://github.com/fabric8-services/keycloak.git --branch master
+  git clone -b 3.2.0.Final  --depth 1 https://github.com/fabric8-services/keycloak.git
```

Comment:
Here's some context about how we arrived at this : https://github.com/fabric8-services/keycloak/pull/93#issuecomment-314536967

But I feel slightly stronger about having the branch deployed into production, that way.. if we push to https://github.com/fabric8-services/keycloak-deployment 's master, we needn't end up deploying changes in https://github.com/fabric8-services/keycloak 's master.